docs/source/transforms.rst

```diff
@@ -121,6 +122,11 @@ Geometry
     v2.RandomHorizontalFlip
     RandomVerticalFlip
     v2.RandomVerticalFlip
+    v2.RandomIoUCrop
+    v2.ScaleJitter
+    v2.RandomShortestSize
+    v2.RandomResize
```

Comment:
Nit: Should we put `RandomIoUCrop` next to the cropping transofrms, and the 3 other ones below (ScaleJitter, RandomShorestestSize and RandomResize) close to `Resize` ?

Reply:
Yes, good point!